README.md

```diff
@@ -31,19 +31,26 @@ Checkstyle can be enabled in Java projects with Maven using the [Maven Checkstyl
         </execution>
     </executions>
     <configuration>
-        <configLocation>
-          https://raw.githubusercontent.com/duraspace/codestyle/master/duraspace-checkstyle.xml
-        </configLocation>
-        <suppressionsLocation>
-          https://raw.githubusercontent.com/duraspace/codestyle/master/duraspace-checkstyle-suppressions.xml
-        </suppressionsLocation>
+        <configLocation>duraspace-checkstyle/checkstyle.xml</configLocation>
+        <suppressionsLocation>*your-checkstyle-suppression.xml*</suppressionsLocation>
```

Comment:
If we are including a suppressions file in the repo, why not include it here, perhaps with a comment above that this can be replaced with your local project suppressions file if you would like to make a change.